### PR TITLE
log: fix some functions comments

### PR DESCRIPTION
**Pull request by BabyHalimao**

### CODE REVIEW

File: log/handler_glog.go [hunk 1]
```diff
@@ -168,8 +168,8 @@ func (h *GlogHandler) WithGroup(name string) slog.Handler {
 	panic("not implemented")
 }
 
-// Log implements Handler.Log, filtering a log record through the global, local
-// and backtrace filters, finally emitting it if either allow it through.
+// Handle implements slog.Handler.Handle, filtering a log record through the global, local
```
Comment (rjl493456442) [inline log/handler_glog.go:171]: Please fix this comment:

```
Handle implements slog.Handler, filtering a log ...
```

Comment (BabyHalimao) [reply log/handler_glog.go:171]: Hi sir, did you mean that I should update whole comments to illustrate the function of `func Handle`?

Comment (AaronChen0) [reply log/handler_glog.go:171]: I think he actually asks to change this:
`slog.Handler.Handle`
=>
`slog.Handler`

Comment (BabyHalimao) [reply log/handler_glog.go]: @rjl493456442 Hi sir, I have updated the function comments, thanks for your review~

**Review by rjl493456442 — APPROVED** [review]

**Review by holiman — COMMENTED** [review]
LGTM

**Review by holiman — APPROVED** [review]

**Pull request merged**